### PR TITLE
J166: * style for fix description width

### PR DESCRIPTION
**Pull request by EugeniusGR**
BEFORE:
image

AFTER
image

### CODE REVIEW

File: frontend/src/components/episode/styles.module.scss [hunk 1]
```diff
@@ -78,6 +87,7 @@
   margin: 0 0 35px 0;
   line-height: 1.25;
   letter-spacing: 1px;
+  width: 75%;
```
Comment (EugeniusGR) [inline frontend/src/components/episode/styles.module.scss]: there is the change

Comment (EugeniusGR) [reply frontend/src/components/episode/styles.module.scss]: forgot to turn off the prettier

Comment (what1s1ove) [inline frontend/src/components/episode/styles.module.scss]: add padding right of the width of the button

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by EugeniusGR** [comment]
image

**Comment by EugeniusGR** [comment]
[Fix description in episodes](https://trello.com/c/OQfrnT0g/166-fix-description-in-episodes)

File: frontend/src/components/episode/episode.tsx [hunk 2]
```diff
@@ -98,10 +103,7 @@ const Episode: React.FC = () => {
                       to={`${AppRoute.PODCASTS}/${episode.podcastId}${AppRoute.EPISODES_EDIT}/${episode.id}`}
                       className={styles.editLink}
                     />
```
Comment (what1s1ove) [inline frontend/src/components/episode/episode.tsx]:
```suggestion
                    >
                      <span className="visually-hidden">Edit episode</span>
                    </Link>
```

File: frontend/src/components/episode/episode.tsx [hunk 3]
```diff
@@ -98,10 +103,7 @@ const Episode: React.FC = () => {
...
-                    <Button
-                      label="Delete"
-                      type={ButtonType.BUTTON}
-                      buttonColor={ButtonColor.LIGHT_PINK}
+                    <button
                       onClick={handleDeleteEpisode}
                       className={styles.deleteButton}
                     />
```
Comment (what1s1ove) [inline frontend/src/components/episode/episode.tsx]:
```suggestion
                    >
                      <span className="visually-hidden">Delete episode</span>
                    </button>
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/episode/episode.tsx (hunk 2 above)
Comment (what1s1ove) [reply frontend/src/components/episode/episode.tsx]: don't fixed

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**